test/test.ts

```diff
@@ -20,6 +25,7 @@ const defaultOptions: Omit<PrettierOptions, keyof ParserOptions> = {
 	importTypeOrder: [IMPORT_TYPE.ALL],
 	packageJSONFiles: ['package.json'],
 	sortingMethod: SORTING_TYPE.LINE_LENGTH,
+	sortingOrder: SORTING_ORDER.DESCENDING,
```

Comment:
Can you add a test case for this? Shouldn't be too hard, just copy one of the alphabetical ones, change the options and invert the order of the expected sorts